includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -114,6 +114,10 @@ public function ensure_learning_mode_url_prefix() {
 			$url = Sensei_Course_Theme::instance()->get_theme_redirect_url( $url );
 		}
 
+		if ( ! empty( $_SERVER['QUERY_STRING'] ) ) {
```

Comment:
It was added to keep the query string when redirecting the user from/to '/learn/'